hikari/commands.py

```diff
@@ -146,6 +146,12 @@ class CommandOption:
     If `builtins.None`, then all channel types will be accepted.
     """
 
+    min_value: typing.Optional[float] = attr.field(default=None, repr=False)
+    """The minimum value permitted (inclusive)"""
+
+    max_value: typing.Optional[float] = attr.field(default=None, repr=False)
```

Comment:
```suggestion
    min_value: typing.Union[int, float, None] = attr.field(default=None, repr=False)
    """The minimum value permitted (inclusive)"""

    max_value: typing.Union[int, float, None] = attr.field(default=None, repr=False)
```
As well as documentation for the `int`/`float` behavior would be nice